packages/ui-kit/src/components/AccessTokenProvider/useAccessToken.ts

```diff
@@ -0,0 +1,16 @@
+import { useContext } from "react"
+
+import { AccessTokenContext } from "./AccessTokenProvider"
+
+interface UseAccessToken {
+  accessToken?: string
+  isLoading: boolean
+  onExpiredToken: () => void
+  failedRetry: boolean
+}
```

Comment:
Document these?

Reply:
Also, if this is identical to AccessTokenContextValue, maybe just have a single interface definition?

Reply:
Right, I'll only use the one from the provider